src/Parallel/Algorithms/AlgorithmArray.ci

```diff
@@ -0,0 +1,48 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+module AlgorithmArray {
+  include "Utilities/TaggedTuple.hpp";
+  include "Parallel/GlobalCache.decl.h";
+
+  // note: the array index here is a hack to use our own types in the chare
+  // array index. The way the ci file is parsed as of charm++ 6.10.2 is such
+  // that:
+  // - the array index type in `AlgorithmArray.decl.h` prepends "CkArrayIndex"
+  //   to what appears in the brackets, which then becomes
```

Comment:
If I'm reading this right you mean `square brackets`. Would you mind adding that clarification since there is also the `SpectreArraindIndex` in the angle brackets?